### PR TITLE
[OSDOCS- 8739]/[OSDOCS-7754] Updating "Creating a cluster on GCP with CCS" and "Creating a cluster on GCP with Google Cloud Marketplace" Docs

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.14+
Issue:

https://issues.redhat.com/browse/OSDOCS-8739: 
Image below shows what was copied from the "Creating a cluster on GCP with CCS" topic and added to the "Creating a cluster on GCP with Google Cloud Marketplace" topic.
image

https://issues.redhat.com/browse/OSDOCS-7754
**(Go to step 6 for review of specific changes based on 7754. Only step 6 and substeps a-c were added for this JIRA ticket).**

Link to docs preview:

https://68003--docspreview.netlify.app/openshift-dedicated/latest/osd_install_access_delete_cluster/creating-a-gcp-cluster#osd-create-gcp-cluster-ccs_osd-creating-a-cluster-on-gcp
https://68003--docspreview.netlify.app/openshift-dedicated/latest/osd_install_access_delete_cluster/creating-a-gcp-cluster#osd-create-cluster-gcp-account_osd-creating-a-cluster-on-gcp

QE review:
- [x] QE has approved this change.

**QE has approved changes regarding https://issues.redhat.com/browse/OSDOCS-7754. Waiting for approval for changes regarding https://issues.redhat.com/browse/OSDOCS-8739.**
**Update: https://issues.redhat.com/browse/OSDOCS-8739 has been QE reviewed.**
Additional information:

**Hello Reviewers!** 
This PR covers two tickets: 
https://issues.redhat.com/browse/OSDOCS-8739: This ticket is in regards to adding the information based on this ticket (https://issues.redhat.com/browse/OSDOCS-6173) to the "Creating a cluster on GCP with Google Cloud Marketplace" topic.  It should have been added to that topic for 6173 but was missed. During the course of working on 8739, it was discovered that there were several steps that in the docs that were no longer valid, so they were removed as well. 
https://issues.redhat.com/browse/OSDOCS-7754: This ticket is in regards to documenting the steps for customer-managed encryption keys (CMK). This info was added to both topics listed in the PR title. 

**Please reach out to me via Slack if you have any questions about the scope of this PR. Thanks!**

### CODE REVIEW

**Comment by mletalie** [comment]
image

**Comment by tzvatot** [comment]
LGTM

**Comment by svmrh** [comment]
The steps for installing OSD into a Shared VPC on Google Cloud looks okay. @jianli-wei - can you please double check it everything looks okay from QE?

The steps for configuring custom KMS keys is already reviewed by QE and OCM Engineering team. Since this is agnostic to the subscription-type/billing model, it is rightfully repeated for both CCS and GCP Marketplace deployment modes. @tzvatot @yingzhanredhat - can you please confirm whether the configuration and steps for custom KMS keys also apply for "[Creating a cluster on GCP with a Red Hat cloud account](https://docs.openshift.com/dedicated/osd_install_access_delete_cluster/creating-a-gcp-cluster.html#osd-create-aws-cluster-ccs_osd-creating-a-cluster-on-gcp)"?

File: modules/osd-create-cluster-ccs.adoc [hunk 1]
```diff
@@ -135,19 +135,35 @@ endif::osd-on-gcp[]
 .. Select a cloud provider region from the *Region* drop-down menu.
 .. Select a *Single zone* or *Multi-zone* configuration.
 .. Leave *Enable user workload monitoring* selected to monitor your own projects in isolation from Red Hat Site Reliability Engineer (SRE) platform metrics. This option is enabled by default.
+ifdef::osd-on-gcp[]
+. Optional: Expand *Advanced Encryption* to make changes to encryption settings.
+
+.. Select *Use Custom KMS keys* to use custom KMS keys. If you prefer not to use custom KMS keys, leave the default setting *Use default KMS Keys*.
++
+With *Use Custom KMS keys* selected:
```
Comment (jianli-wei) [inline modules/osd-create-cluster-ccs.adoc:143]: With **Use Custom KMS keys** selected, the IAM service account `osd-ccs-admin` needs to be granted the role **Cloud KMS CryptoKey Encrypter/Decrypter** to the KMS key (e.g. see [Google Cloud Key Management Service guides](https://cloud.google.com/kms/docs/iam#kms-add-iam-member-gcloud)), so suggest to add a note telling it, thanks!

**Review by jianli-wei — CHANGES_REQUESTED** [review]

**Comment by mletalie** [comment]
> The steps for installing OSD into a Shared VPC on Google Cloud looks okay. @jianli-wei - can you please double check it everything looks okay from QE?
> 
> The steps for configuring custom KMS keys is already reviewed by QE and OCM Engineering team. Since this is agnostic to the subscription-type/billing model, it is rightfully repeated for both CCS and GCP Marketplace deployment modes. @tzvatot @yingzhanredhat - can you please confirm whether the configuration and steps for custom KMS keys also apply for "[Creating a cluster on GCP with a Red Hat cloud account](https://docs.openshift.com/dedicated/osd_install_access_delete_cluster/creating-a-gcp-cluster.html#osd-create-aws-cluster-ccs_osd-creating-a-cluster-on-gcp)"?

FYI: Based on my conversation with [@priya](https://redhat.enterprise.slack.com/team/US21AESBT) and my own testing, config and steps for KMS keys would not apply to the "Creating a cluster on GCP with a Red Hat cloud account" topic.

File: modules/osd-create-cluster-ccs.adoc (hunk 1 above)
Comment (mletalie) [reply modules/osd-create-cluster-ccs.adoc:148]: Thanks @jianli-wei, info has been added. Thanks for the QE review!

**Comment by bmcelvee** [comment]
LGTM!

**Pull request merged**